packages/shared/src/components/filters/FeedFilters.tsx

```diff
@@ -1,21 +1,17 @@
 import React, { ReactElement, useState } from 'react';
-import classNames from 'classnames';
-import { ModalProps, StyledModal } from '../modals/StyledModal';
```

Comment:
I think this file can now be deleted right?

Reply:
Let me double-check it quickly.

Reply:
We still have the `StyledModal` used in the `SteppedModal` which is used by the onboarding modal.

Reply:
I see but that's also related to feed filters no?
It's fine for this scope, as long as we address that one as well 🙏

Reply:
@el do we already have one variant that has a progress bar like the onboarding modal?

Reply:
Amazing. Let me try using that one.